.github/workflows/jira-issue-creator.yml

```
@@ -0,0 +1,22 @@
+# Creates jira tickets for new github issues to help triage
+name: Jira Issue Creator For GTM template
+
+on:
+  issues:
+    types: [opened]
+  workflow_call:
+    inputs:
+      label:
+        type: string
+
+jobs:
+  call-workflow-passing-data:
+    uses: amplitude/amplitude-browser-sdk-gtm-template/.github/workflows/jira-issue-create-template.yml@main
```

Reuse existing template?
```suggestion
    uses: amplitude/Amplitude-TypeScript/.github/workflows/jira-issue-create-template.yml@main
```